docs/source/deploy_chaincode.md

```diff
@@ -0,0 +1,481 @@
+```
+
+The transaction above uses the `--peerAddresses` flag to target `peer0.org1.example.com` from Org1 and `peer0.org2.example.com` from Org2. The `commit` transaction is submitted to the peers joined to the channel to query the chaincode definition approved the the organization that operates the peer. The command needs to target the peers from a sufficient number of organizations to meet the policy for deploying a chaincode. Because the approval is distributed within each organization, you can target any peer that belongs to a channel member.
+
+After the transaction queries the parameters approved by channel members, the transaction is then submitted to the ordering service to be added to a block and distributed to the channel. The peers on the channel then validate whether a sufficient number of organizations have approved the chaincode definition. The `peer lifecycle chaincode commit` command will wait for the validations from the peer before returning a response.
+
+You can use the [peer lifecycle chaincode querycommitted](commands/peerlifecycle.html#peer-lifecycle-chaincode-querycommitted) command to confirm that the chaincode definition has been committed to the channel.
+```
+peer lifecycle chaincode querycommitted --channelID mychannel --name fabcar --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+```
+If the chaincode was successful committed to the channel, the `querycommitted` command will return the sequence and version of the chaincode definition:
+```
+Committed chaincode definition for chaincode 'fabcar' on channel 'mychannel':
+Version: 1, Sequence: 1, Endorsement Plugin: escc, Validation Plugin: vscc, Approvals: [Org1MSP: true, Org2MSP: true]
+```
+
+## Invoking the chaincode
+
+After the chaincode definition has been committed to a channel, the chaincode will start on the peers joined to the channel where the the chaincode was installed. The chaincode is then ready to be invoked by client applications. Because we included the ``--init-required`` flag in the chaincode definition, the first transaction needs invoke the Init function to initialize the chaincode. Issue the following command to invoke Init. Note that the invoke command needs to contain the ``--isInit`` flag and target a sufficient number of peers to meet chaincode endorsement policy.
```

Comment:
Do you "invoke" init? Actually asking --- not sure what the proper term for this would be.

Reply:
Invoke Init makes sense because you need to use the `peer invoke` command.

Reply:
this transaction is also now a normal transaction - so doesn't have the restrictions as to what can do.